### DISCUSSION TITLE
Istio Operator

### DISCUSSION THREAD

**Discussion by MarkJack26** [istio/istio#52793] [Q&A]
Hi all,

Does the Istio Operator require cluster admin and is there documentation to back this up. It has been flagged as part as a PenTest.

Thanks

Mark.

**Comment by howardjohn** [comment]
Yes it does, which is part of the reason it has been discouraged for 3 years, and has now been official deprecated and slated for removal in the next release: https://istio.io/latest/blog/2024/in-cluster-operator-deprecation-announcement/.

It is recommended you do not use the operator and instead migrate to `istioctl install` or Helm, whichever meets your needs best.

**Comment by MarkJack26** [comment]
Hi John,

Thanks a lot for this.

Regards

Mark.

On Wed, 21 Aug 2024 at 16:22, John Howard ***@***.***> wrote:

> Yes it does, which is part of the reason it has been discouraged for 3
> years, and has now been official deprecated and slated for removal in the
> next release:
> https://istio.io/latest/blog/2024/in-cluster-operator-deprecation-announcement/
> .
>
> It is recommended you do not use the operator and instead migrate to istioctl
> install or Helm, whichever meets your needs best.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/istio/istio/discussions/52793#discussioncomment-10409492>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFKC6APIDDLQOZGDRJCV7HTZSSWCDAVCNFSM6AAAAABM4IL7HOVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANBQHE2DSMQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
[image:
https://www.youracclaim.com/badges/a49d4a8b-9c77-461f-b802-01beac8470fb/public_url]
<https://www.youracclaim.com/badges/a49d4a8b-9c77-461f-b802-01beac8470fb/public_url>
